python/semantic_kernel/connectors/memory/qdrant/qdrant_config.py

```diff
@@ -0,0 +1,4 @@
+import os
```

Comment:
What is this config being used for?

Reply:
Addressed in https://github.com/microsoft/semantic-kernel/pull/919/commits/e062fa799c4a38153cc809cf84f15c250977301f